packages/functions/lambdas/dbOperations.ts

```diff
@@ -0,0 +1,28 @@
+import { executeSql } from "@/db/queries/dbOperations";
+import { errorResult, okResult } from "@/utils/lambda/result";
+
+export const handler = async (event: { sql: string }) => {
```

Comment:
- Would this work in production? This is extremely unsafe. This should only exist in test ci mode.

Reply:
This would cause nothing, if the tests are not running in production, this lambda is not exposed, via API gateway